src/main/scala/nl/knaw/dans/api/sword2/StatementManagerImpl.scala

```diff
@@ -38,8 +39,18 @@ class StatementManagerImpl extends StatementManager {
       case Success(properties) =>
         val statement = new AtomStatement(iri, "DANS-EASY", s"Deposit ${SwordID.extract(iri).get}", properties.timeStamp)
         statement.setState(properties.label, properties.description)
+        if (properties.label == State.ARCHIVED.toString)
+          properties.resources.foreach(resources => setResources(statement, resources))
         statement
       case Failure(t) => throw new SwordError(404)
     }
   }
+
+  private def setResources(statement: AtomStatement, resources: PropertiesResources): Unit =  {
+    val bagIdResource = new ResourcePart(resources.bagId.trim)
+    bagIdResource.setMediaType("multipart/related;type=bag")
+    val filePathsResources = resources.filePaths.map(resource => { val resourcePart = new ResourcePart(resource.trim); resourcePart.setMediaType("multipart/related;type=file"); resourcePart})
```

Comment:
not entirely sure about this lambda... I don't think you need the `resourcePart`